packages/components/src/components/hds/table/th-selectable.hbs

```diff
@@ -3,14 +3,29 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
-<Hds::Table::Th class="hds-table__th--is-selectable" @scope={{@selectionScope}} ...attributes>
-  <Hds::Form::Checkbox::Base
-    id={{this.checkboxId}}
-    class="hds-table__checkbox"
-    checked={{@isSelected}}
-    aria-label={{this.ariaLabel}}
-    {{did-insert this.didInsert}}
-    {{will-destroy this.willDestroyNode}}
-    {{on "change" this.onSelectionChange}}
-  />
+<Hds::Table::Th
+  class="hds-table__th--is-selectable"
+  aria-sort={{this.ariaSort}}
+  @scope={{@selectionScope}}
+  ...attributes
+>
+  <div class="hds-table__th-content">
+    <Hds::Form::Checkbox::Base
+      id={{this.checkboxId}}
+      class="hds-table__checkbox"
+      checked={{@isSelected}}
+      aria-label={{this.ariaLabel}}
+      {{did-insert this.didInsert}}
+      {{will-destroy this.willDestroyNode}}
+      {{on "change" this.onSelectionChange}}
+    />
+    {{#if this.isSortable}}
+      <span id={{this.labelId}} class="sr-only">selection state</span>
```

Comment:
Should we make this the dynamic sort key? Probably not meaningful to the end user.

Reply:
for consistency with the other similar implementation, probably leaving the `labelId` makes sense

Reply:
@didoo You mean like this?

`<span id={{this.labelId}} class="sr-only">{{this.labelId}}</span>`